lib/experian/client.rb

```diff
@@ -49,7 +49,7 @@ def excon_options
     end
 
     def request_uri
-      Experian.net_connect_uri
+      raise "Request uri undefined"
```

Comment:
Do we want to `raise` an exception here? that would make the `post_request` fail?